### PR TITLE
Invite fragment basics

### PR DESCRIPTION
**Pull request by bensouthern**
The basics of the invite fragment.  Just the core structure

### CODE REVIEW

File: app/src/main/res/layout/list_item_hist.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
             android:text="@string/hist_btn_contact_courier"
```
Comment (DerekFei) [inline app/src/main/res/layout/list_item_hist.xml]: There are two file list_item_hist and list_item_hist2  could you give them a more clear name? thx

File: app/src/main/java/com/packants/android/Adapters/InvitePendingListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.packants.android.R;
+
+import java.util.ArrayList;
+
+/**
+ * Created by packants on 04/08/16.
+ */
```
Comment (DerekFei) [inline app/src/main/java/com/packants/android/Adapters/InvitePendingListAdapter.java]: delete this comment

File: app/src/main/java/com/packants/android/Adapters/InvitePendingListAdapter.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+    @Override
+    public int getCount() {
+        return contacts;
+    }
+
+    @Override
+    public Object getItem(int i) {
+
+
```
Comment (DerekFei) [inline app/src/main/java/com/packants/android/Adapters/InvitePendingListAdapter.java]: white space

File: app/src/main/java/com/packants/android/Adapters/InviteSuccessListAdapter.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.packants.android.R;
+
+import java.util.ArrayList;
+
+/**
+ * Created by packants on 04/08/16.
+ */
```
Comment (DerekFei) [inline app/src/main/java/com/packants/android/Adapters/InviteSuccessListAdapter.java]: delete this comment

File: app/src/main/java/com/packants/android/Adapters/InviteSuccessListAdapter.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+    @Override
+    public int getCount() {
+        return contacts;
+    }
+
+    @Override
+    public Object getItem(int i) {
+
+
```
Comment (DerekFei) [inline app/src/main/java/com/packants/android/Adapters/InviteSuccessListAdapter.java]: white space

File: app/src/main/java/com/packants/android/Adapters/HistoryListAdapter.java [hunk 6]
```diff
@@ -34,8 +34,8 @@
     private View headerView;
     private HistoryFragment historyFragment;
 
-    public ExpandableListAdapter(Context _context, List<String> _listDataHeader,
-                                 HashMap<String, List<String>> _listChildData, HistoryFragment _historyFragment) {
+    public HistoryListAdapter(Context _context, List<String> _listDataHeader,
```
Comment (0xABC) [inline app/src/main/java/com/packants/android/Adapters/HistoryListAdapter.java]: Underscore should not be necessary, and I don’t think we’ve been using this convention anywhere else.

File: app/src/main/java/com/packants/android/Fragments/InviteFragment.java [hunk 7]
```diff
@@ -20,13 +30,147 @@ public InviteFragment() {
...
+                if (successContents.getVisibility() != View.VISIBLE) {
+                    successContents.setVisibility(View.VISIBLE);
+                } else if (successContents.getVisibility() != View.GONE) {
+                    successContents.setVisibility(View.GONE);
+                }
+            }
+        });
+    }
+
+    public void setHeaderSection() {
```
Comment (0xABC) [inline app/src/main/java/com/packants/android/Fragments/InviteFragment.java]: any reason these are public?

**Comment by 0xABC** [comment]
Other than my comments looks ok to me.  

The UI may need some work though, it might be a bit confusing for the user. When testing on my device, initially the pending list was collapsed but the successful one expanded - might be more clear to start with both collapsed. Also the nested scrolling could be problematic, but I’m not sure of a better solution at the moment (perhaps tabs for pending and successful?).

**Pull request merged**